Library/ViewModels/SettingsAccountViewModel.swift

```diff
@@ -88,6 +88,12 @@ SettingsAccountViewModelOutputs, SettingsAccountViewModelType {
 
     self.viewDidAppearProperty.signal
       .observeValues { _ in AppEnvironment.current.koala.trackAccountView() }
+
+    // Koala
+    updateCurrency.signal
+      .observeValues { currency in
```

Comment:
Can we use 1 liner here? `.observeValues { AppEnvironment.current.koala.trackChangedCurrency($0) }` or is it better to be explicit?